sys/shell/commands/sc_vfs.c

```diff
@@ -172,6 +180,35 @@ static int _mount_handler(int argc, char **argv)
     return res;
 }
 
+static int _umount_handler(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("usage: %s [path]\n", argv[0]);
+        puts("umount pre-configured mount point");
+        return -1;
+    }
+
+    int res = vfs_unmount_by_path(argv[1]);
+    puts(strerror(res));
```

Comment:
Is `_errno_string` less expensive than `strerror`?
I know I used it for `_mount_handler` but that was probably a mistake given how much ROM it adds.